section2/reflection.md

```diff
@@ -1,15 +1,26 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+* You should never really compare yourself to anyone else - it is important to track your own progress from the beginning to see and remind yourself of how far you've come. When you track your own progress, it is less daunting than comparing yourself to where other people might be in their own journey.
+
+* Reminding yourself in moments where you're struggling of "what do I do now?" Instead of panicking about not knowing something, recognize you do not know it and start reading about it to try and understand it. Shift the reaction from panic to "Ok I do not know it, what do I do to fix it."
 
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+* I currently demonstrate a growth mindset by believing that I am capable of learning a new set of skills through persistence and effort.
+* I do not demonstrate a growth mindset by getting too hung up on my "failures" sometimes. Instead, I need to view them as opportunities for continued learning and growth and not as a reflection of who I am as an individual. I also have a tendency to compare myself to other people and that is not helpful at all as everyone works and learns at a different pace.
 
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+* At first it gave me a bit of anxiety because I was like "Oh no I'm supposed to know this!" But then I take a breath and remind myself I am capable and I am learning, and do the necessary research to check I am following the proper directions. This supported my learning by proving I am capable of figuring things out and that panic is not necessary.
```

Comment:
YES you are capable of figuring things out!!! Keep reminding yourself of that

MOST things - you aren't supposed to know. You are just supposed to be able to use your tools to figure it out.